setup/teardown.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Teardown
+description: Learn how to teardown Coder and its associated infrastructure.
+---
+
+This guide shows you how to teardown Coder and the cluster it is deployed on.
+
+**Note:** these instructions help you remove infrastructure spun up via our
+[Kubernetes Documentation](https://coder.com/docs/setup/kubernetes), and do not
+include teardown steps for any additional cluster resources that may have been deployed.
+
+## Amazon Elastic Kubernetes Service (EKS)
+
+1. Make sure you're running `eksctl` version 0.37.0 or later by running
+the following command:
+
+```bash
+eksctl version
+```
+
+2. List all services in the cluster by running the following command:
+
+```bash
+kubectl get svc --all-namespaces
+```
+
+3. Delete any services that have an `EXTERNAL-IP` value
+by running the following command:
+
+```bash
+kubectl delete svc <service-name>
+```
```

Comment:
This needs an additional step:

You need to ensure `kubectl` is pointed to the same context still. Add steps to check and set the context to the right cluster before you run the delete command, or at least info to make sure people aren't blinding deleting services.

Maybe something like this would work:

```suggestion
3. Delete any services that have an `EXTERNAL-IP` value
in your namespace by running the following command:

```bash
kubectl delete svc <service-name>
```
```